src/charity_base.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# Taken from their website's "What is CharityBase?"
+description: >
+  Charity Base is an API that aggregates data from the 168,000 charities
+  registered in England and Wales.
```

Comment:
The project sounds great! Do you know if they have plan to extend the project outside of Britain? I couldn't figure this out at a glance

Reply:
They do plan to add Scottish and Northern Irish charities, too: https://charitybase.uk/about/other-countries
I can't tell whether they plan to expand further into europe or make it so that you can better add new countries, but they also mention that you can send them an email if you want to add other countries.

Reply:
Oh, that's good! Sounds more friendly this way :) Definitely a good opportunity for some folks out there :)

Reply:
Should I maybe add `and potentially other countries` to the description?
To emphasis that the authors generally (presumably) support adding other countries?

E.g.:

> Charity Base is an API that aggregates data from the 168,000 charities registered in England and Wales - potentially expanding to other countries in the future.

Reply:
Yes, definitely! We want to motivate our users